src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+}
+
+.searchBar {
+  display: inline-block;
+  width: 100%;
+  font-family: Avenir-Book, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+}
+
+.searchBar--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+
+  background-position-x: 26px;
+  background-size: 19px;
+}
+
+.searchBar--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+
+  background-position-x: 13px;
+  background-size: 11px;
+}
+
+.searchBar--big::placeholder {
+  font-family: Avenir-Book, sans-serif;
+}
+
+.searchBar--small::placeholder {
+  font-family: Avenir-Book, sans-serif;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar--big:focus {
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 3px;
+  outline: none;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.searchBar--small:focus {
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 3px;
+  outline: none;
```

Comment:
generally, if you remove the outline, you need to set a special border on focus, so a user has an idea of which search bar is selected. here the background changes, but just remember it for the future